spec/features/mailing_list_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+RSpec.describe "Mailing list", type: :request do
+  describe "/mailing-list" do
```

Comment:
I dont think this is needed because we already mention that we describing how mailing list route works. 

I also think this should be under "request" folder rather than features. The reason is that this is a request spec and its not going through a full journey like the other feature tests i.e load up a page, check the content and then click on link etc.  This is an odd one because we dont have a controller for this route but we are just test how routes are setup and I'd expect to see those sort of tests under requests.

Reply:
Ah woops yes I kept changing my mind on whether it should be a request spec or feature spec and ended up doing both somehow haha. I'll move it to the correct folder.

Reply:
Done.